query-connector/src/app/(pages)/userManagement/page.tsx

```diff
@@ -4,22 +4,128 @@ import { useContext, useEffect, useState } from "react";
 import { Button } from "@trussworks/react-uswds";
 import classNames from "classnames";
 import Table from "../../ui/designSystem/table/Table";
-import { User, UserGroup, usersMock } from "./types";
 import RoleDropdown from "./components/RoleDropdown";
 import { UserManagementContext } from "./components/UserManagementProvider";
+import {
+  RoleTypeValues,
+  User,
+  UserGroup,
+} from "../../models/entities/user-management";
+import { getUsers, updateUserRole } from "@/app/backend/user-management";
+import { QCResponse } from "@/app/models/responses/collections";
+import { showToastConfirmation } from "@/app/ui/designSystem/toast/Toast";
+import { useSession } from "next-auth/react";
 
 /**
  * User section in the user management page
  * @returns Users table
  */
 const UserManagement: React.FC = () => {
-  const [users, setUsers] = useState<User[]>([]);
+  const { data: session } = useSession();
+  console.log("user management session:", session);
```

Comment:
@katyasoup after this PR is merged the user's role will be available in the session object